dags/pytorch_xla/r2_3.py

```diff
@@ -0,0 +1,216 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+from airflow.decorators import task_group
+from airflow import models
+from xlml.apis import gcp_config, metric_config, task, test_config
+from dags import composer_env
+from dags.vm_resource import Project, Zone, V5_NETWORKS, V5E_SUBNETWORKS
+
+
+# Run once a day at 2 pm UTC (6 am PST)
+SCHEDULED_TIME = None
```

Comment:
Can you update this to once per day?

Reply:
As @will-cromar suggested, we set `None` here and we can trigger manually. tests for rc doesn't need to be run daily, since they tag is immutable

Reply:
Oh, I see, make sense to me!

Reply:
Shouldn't we only run this manually when the wheel updates?